### PR TITLE
out_azure_kusto: added-kusto-specific-headers

### PR DESCRIPTION
**Pull request by tanmaya-panda1**
Signed-off-by: Tanmaya Panda <tanmayapanda@microsoft.com>
(cherry picked from commit ccfc8ee8bb261107b14c442b8687966ede4fd1bd)

----
Enter `[N/A]` in the box, if an item is not applicable to your change.

**Testing**
Before we can approve your change; please submit the following in a comment:

- [ ] Example configuration file for the change
- [ ] Debug log output from testing the change

- [ ] Attached [Valgrind](https://valgrind.org/docs/manual/quick-start.html) output that shows no leaks or memory corruption was found

If this is a change to packaging of containers or native binaries then please confirm it works for all targets.

- [ ] Run [local packaging test](./packaging/local-build-all.sh) showing all targets (including any new ones) build.
- [ ] Set `ok-package-test` label to test for all targets (requires maintainer to do).

**Documentation**

- [ ] Documentation required for this feature

**Backporting**

- [ ] Backport to latest stable release.

----

Fluent Bit is licensed under Apache 2.0, by submitting this pull request I understand that this code will be released under the terms of that license.

### CODE REVIEW

File: plugins/out_azure_kusto/azure_kusto_ingest.c [hunk 1]
```diff
@@ -164,6 +165,12 @@ static flb_sds_t azure_kusto_create_blob(struct flb_azure_kusto *ctx, flb_sds_t
                 flb_http_add_header(c, "x-ms-date", 9, tmp, len);
                 flb_http_add_header(c, "x-ms-version", 12, "2019-12-12", 10);
 
+                //added kusto specific headers for debugging requests
```
Comment (edsiper) [inline plugins/out_azure_kusto/azure_kusto_ingest.c:168]: please use /* */

File: plugins/out_azure_kusto/azure_kusto_ingest.c [hunk 2]
```diff
@@ -367,6 +374,12 @@ static int azure_kusto_enqueue_ingestion(struct flb_azure_kusto *ctx, flb_sds_t
                     flb_http_add_header(c, "x-ms-date", 9, tmp, len);
                     flb_http_add_header(c, "x-ms-version", 12, "2019-12-12", 10);
 
+                    //added kusto specific headers for debugging requests
```
Comment (edsiper) [inline plugins/out_azure_kusto/azure_kusto_ingest.c:377]: please use /* */

**Review by edsiper — CHANGES_REQUESTED** [review]

**Comment by tanmaya-panda1** [comment]
@edsiper : request your review on this PR

**Comment by tanmaya-panda1** [comment]
Since these are kusto headers which are needed for tracking requests in kusto, we should not need any additional documentation update since it has no impact on the customer.

**Pull request closed without merging**